pkg/defaultcomponents/defaults.go

```diff
@@ -68,10 +72,21 @@ import (
 	"go.uber.org/multierr"
 )
 
+var fileLogReceiverFeatureGate = featuregate.GlobalRegistry().MustRegister("adot.filelog.enabled",
```

Comment:
```suggestion
var fileLogReceiverFeatureGate = featuregate.GlobalRegistry().MustRegister("adot.filelog",
```
I believe putting the `.enabled` in the feature gate name is not necessary and possibly confusing. The enabling of a feature gate is defined by it's stage and also whether or not the user provides `+` or `-` at collector startup. 

I also wonder if we should consider namespacing this feature gate further. `adot.receiver.filelog`?

Reply:
Sure I can namespace them like that, thanks for the suggestion.

Reply:
Or actually, to be consistent with how these components are named in the opentelemetry contrib repo, I'll namespace them like `adot.filelog.receiver`

Reply:
Resolving this conversation as I'm assuming my suggestion for the namespace change is fine.